src/frontend/Komponenter/Behandling/BehandlingContainer.tsx

```diff
@@ -19,26 +19,37 @@ import { HenleggModal } from './Henleggelse/HenleggModal';
 
 const Container = styled.div`
     display: flex;
+
+    flex-shrink: 2;
 `;
 
-const HøyreMenyWrapper = styled.div`
+interface InnholdWrapperProps {
+    åpenHøyremeny: boolean;
+}
+
+interface HøyreMenyWrapperProps {
+    åpenHøyremeny: boolean;
+}
+
+const HøyreMenyWrapper = styled.div<HøyreMenyWrapperProps>`
     border-left: 2px solid ${navFarger.navGra40};
-    overflow-x: hidden;
 
-    width: 20rem;
+    flex-shrink: 1;
+    flex-grow: 0;
 
-    @media (max-width: 800px) {
-        width: 0rem;
-    }
-    overflow-y: auto;
+    width: ${(p) => (p.åpenHøyremeny ? '20rem' : '1.5rem')};
+    min-width: ${(p) => (p.åpenHøyremeny ? '20rem' : '1.5rem')};
 `;
 
-const InnholdWrapper = styled.div`
-    flex: 1;
-    max-width: calc(100% - 20rem);
-    @media (max-width: 800px) {
-        max-width: 100%;
-    }
+const InnholdWrapper = styled.div<InnholdWrapperProps>`
+    flex-shrink: 0;
+    flex-grow: 1;
+    flex-basis: 0px;
+    min-width: 0px;
+
+    overflow-x: scroll;
+
+    max-width: ${(p) => (p.åpenHøyremeny ? 'calc(100% - 1.5rem)' : '100%')};
```

Comment:
Trenger vi ikke calc(100%-20rem) her for å unngå vertikal skrolling i "normalcase" ??

Reply:
Bra catch! Fikset i [a638fd0](https://github.com/navikt/familie-ef-sak-frontend/pull/944/commits/a638fd0d8b9c4ba4843143f9400abfa6ea92d605)